src/lib/tzcode/tzcode.h

```diff
@@ -37,12 +37,16 @@ struct tnt_tm {
 
 	/** Seconds east of UTC. */
 	long int tm_gmtoff;
-	/** Timezone abbreviation. */
-	const char *tm_zone;
 	/** Seconds since Epoch */
 	int64_t tm_epoch;
 	/** nanoseconds */
 	int tm_nsec;
+#if 0
+	/** Timezone abbreviation. */
+	const char *tm_zone;
+#else
+	int16_t tm_tzindex;
+#endif
```

Comment:
Don't forget to delete the unneeded code.

Reply:
Deleted